website/data/docs-nav-data.json

```diff
@@ -674,7 +674,24 @@
           },
           {
             "title": "Ingress Gateways",
-            "path": "connect/gateways/ingress-gateway"
+            "routes": [
+              {
+                "title": "Overview",
+                "path": "connect/gateways/ingress-gateway"
+              },
+              {
+                "title": "Implement an ingress gateway",
+                "path": "connect/gateways/ingress-gateway/ingress-gateways-usage"
+              },
+              {
+                "title": "Serve TLS certificates from an external service",
+                "path": "connect/gateways/ingress-gateway/ingress-gateways-tls-external-service"
+              },
```

Comment:
The paths here feel redundant to me. Would it make sense to have the following?
- connect/gateways/ingress-gateway/usage
- connect/gateways/ingress-gateway/tls-external-service